### PR TITLE
Merge latest commits from CurationConcerns

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ Metrics/ParameterLists:
 
 Metrics/AbcSize:
   Exclude:
+    - 'app/actors/hyrax/actors/add_to_work_actor.rb'
```
Comment (jcoyne) [inline .rubocop_todo.yml]: 😞

File: app/actors/hyrax/actors/add_to_work_actor.rb [hunk 2]
```diff
@@ -22,13 +22,17 @@ def add_to_works(new_work_ids)
             work.save
           end
 
-          # add to new
+          # add to new so long as the depositor for the parent and child matches, otherwise inject an error
           (new_work_ids - curation_concern.in_works_ids).each do |work_id|
             work = ::ActiveFedora::Base.find(work_id)
-            work.ordered_members << curation_concern
-            work.save
+            if work.depositor != curation_concern.depositor
```
Comment (jcoyne) [inline app/actors/hyrax/actors/add_to_work_actor.rb]: wasn't this updated to use CanCan in CC?

Comment (mjgiarlo) [inline app/actors/hyrax/actors/add_to_work_actor.rb]: It was, and it was merged about ten minutes after I made this PR. I'll try including that commit in this branch but if it gets gnarly, I will be inclined to put in a follow-up PR instead.

File: .rubocop_todo.yml (hunk 1 above)
Comment (mjgiarlo) [inline .rubocop_todo.yml]: I'll take another look at this.

**Comment by mjgiarlo** [comment]
@jcoyne All changes made.

**Comment by mjgiarlo** [comment]
@jcoyne re-review?

**Pull request merged**